### ISSUE TITLE
unable to connect latest version peerjs-server

### ISSUE THREAD

**Issue by lordzpipo** [peers/peerjs-server#124]
Hi,
I am new to peerjs-server and trying to setup a custom peerjs server to integrate with my project.

But when i using latest update of peerjs server, client-side unable connect to server (fully copy sample code).
After few hour's research, i found out `https://www.npmjs.com/package/peer` version is working well.

So is there is any document is not updated or maybe it's bug?

This is error msg when i using latest version:
```
peerjs.min.js:138 GET http://localhost:9000/peerjs/peerjs/id?ts=15571397875680.7781818557504341 404 (Not Found)
```

and this is my server code
```js
var express = require('express');
var app = express();
var ExpressPeerServer = require('peer').ExpressPeerServer;

app.get('/', function (req, res, next) { res.sendFile(`${__dirname}/index.html`); });

var options = {
    debug: true,
}

var server = require('http').createServer(app);
var peerserver = ExpressPeerServer(server, options);

app.use('/peerjs', peerserver);

peerserver.on("connection", (client) => {
    console.log(client);
})

server.listen(9000);
```
client-side
```js
var peer = new Peer('', {host: "localhost", port: 9000, path: "/peerjs"});
peer.on("open", id => {
    console.log(id);
})
```

ps: using `https://www.npmjs.com/package/peer` without change any code is working well.

**Comment by zhou-yg** [comment]
look my pr -> https://github.com/peers/peerjs-server/pull/125/files

Is that providing any help ?

**Comment by andersoxie** [comment]
I get a similar error when running with Docker:

peer.min.js:1 GET http://anders-ubuntu-server:9000/myapp/peerjs/id?ts=15612893884720.787367880667849 404 (Not Found)
o._retrieveId @ peer.min.js:1
o @ peer.min.js:1
initialize @ receive.html:66
(anonymous) @ receive.html:234
(anonymous) @ receive.html:235

Built:

 sudo docker build -t peerjs https://github.com/peers/peerjs-server.git

Started:

sudo docker run -p 9000:9000 -d peerjs

Processes

root     19163  1608  0 Jun22 ?        00:00:00 /usr/bin/docker-proxy -proto tcp -host-ip 0.0.0.0 -host-port 9000 -container-ip 172.17.0.2 -container-port 9000
root     19191 19172  0 Jun22 ?        00:00:04 node bin/peerjs --port 9000 --path /myapp


When trying to connect I get an error so I used curl to see what the server respons.


$ curl http://localhost:9000/myapp

<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>Cannot GET /myapp</pre>
</body>
</html>


As I interpret the answer, the server is running but it does not find /myapp

**Comment by afrokick** [comment]
Fixed in #125
